package.json

```diff
@@ -1,46 +1,65 @@
 {
   "name": "ten99policy",
-  "version": "1.0.5",
-  "description": "1099Policy API wrapper",
+  "version": "9.15.0",
+  "description": "Ten99Policy API wrapper",
```

Comment:
Minor, but you're ok to use 1099Policy here since it's the description, and company name is 1099Policy.